runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -10590,26 +10590,43 @@ static TR::Register* inlineHasNegativesOrCountPositives(TR::Node* node, TR::Reco
    // Length of byte array
    TR::Register *lengthReg = cg->evaluate(node->getChild(2));
 
+   // Store a boolean indicating whether we are generating code for hasNegatives or countPositives
+   bool isHasNegatives = recognizedMethod == TR::java_lang_StringCoding_hasNegatives;
+
+   // A key part of the main loop of this algorithm is the pmovmskb instruction,
+   // which extracts the sign bits from a source register and collects them into a destination register
+   // We need to specify the encoding of this instruction so that the code generator knows we want the 16 byte version
+   TR::InstOpCode pmovmskb = TR::InstOpCode::PMOVMSKB4RegReg;
+   OMR::X86::Encoding pmovmskbEncoding = pmovmskb.getSIMDEncoding(&cg->comp()->target().cpu, TR::VectorLength128);
+   bool useVectorInstructions = (pmovmskbEncoding != OMR::X86::Encoding::Bad) && feGetEnv("TR_disableSIMDHasNegativesCountPositives") == NULL;
```

Comment:
Code in the JIT typically will assign the result of a call to `feGetEnv` to a static variable so that it will be evaluated only once during any JVM instance.

Reply:
```suggestion
   static bool disableSIMDHasNegativesCountPositives = feGetEnv("TR_disableSIMDHasNegativesCountPositives") != NULL
   bool useVectorInstructions = (pmovmskbEncoding != OMR::X86::Encoding::Bad) && !disableSIMDHasNegativesCountPositives;
```